### PR TITLE
tgui: Holopad

### PR DESCRIPTION
**Pull request by Arkatos1**
## About The Pull Request

This PR converts Holopad from HTML based interface to tgui, with some improvements and cleanup of the related code. Comparison images below.

**OLD:**

HolopadOLD

**NEW:**

NewHolopad

## Why It's Good For The Game

Better UI.

## Changelog
:cl: Arkatos
add: Holopad now uses tgui.
/:cl:

### CODE REVIEW

**Comment by stylemistake** [comment]
Makes holopads actually usable

File: tgui/packages/tgui/interfaces/Holopad.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+            textAlign="center"
+            fontSize="24px">
+            <Button
+              lineHeight="40px"
+              icon="times"
+              content="Hang Up"
+              color="bad"
+              onClick={() => act('hang_up')} />
+          </Box>
+        </Fragment>
```
Comment (stylemistake) [inline tgui/packages/tgui/interfaces/Holopad.js:34]: Use a Dimmer here perhaps, it also auto-centers everything.

Comment (Arkatos1) [reply tgui/packages/tgui/interfaces/Holopad.js]: Used a Modal instead.

**Comment by Arkatos1** [comment]
Showcase of Holopad dialing:

Dialing

**Comment by Cyberboss** [comment]
Based and redbilled

**Comment by Cyberboss** [comment]
Also
> Getting other people to improve my horrendously shittily implemented features
image

**Pull request merged**